### PR TITLE
1436790/1438462: Part 48: 20 files with MSDN link fixes.

### PR DESCRIPTION
**Pull request by DennisLee-DennisLee**
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

### CODE REVIEW

**Comment by DennisLee-DennisLee** [comment]
The `git reset` trick didn't work - still lots of commits. :(

**Comment by mairaw** [comment]
Ping me offline via teams and I can work with you to get that working. You might be missing a few extra steps.

File: docs/visual-basic/misc/bc33028.md [hunk 1]
```diff
@@ -26,4 +26,4 @@ A conversion operator is declared with a return type of the [Object Data Type](.
 - [Operator Statement](../../visual-basic/language-reference/statements/operator-statement.md)
 - [How to: Define an Operator](../../visual-basic/programming-guide/language-features/procedures/how-to-define-an-operator.md)
 - [How to: Define a Conversion Operator](../../visual-basic/programming-guide/language-features/procedures/how-to-define-a-conversion-operator.md)
-- [Object as the Universal Data Type (Visual Basic)](https://msdn.microsoft.com/library/5315bf21-2b22-45ab-98cd-5631dffbcb2f)
+- [Object as the Universal Data Type (Visual Basic)](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/3cf69sdx(v=vs.100))
```
Comment (mairaw) [inline docs/visual-basic/misc/bc33028.md]:
```suggestion
- [Object Data Type (Visual Basic)](../language-reference/data-types/object-data-type.md)
```

File: docs/visual-basic/misc/bc33032.md [hunk 2]
```diff
@@ -26,4 +26,4 @@ A conversion operator is declared with a parameter of the [Object Data Type](../
 - [Operator Statement](../../visual-basic/language-reference/statements/operator-statement.md)
 - [How to: Define an Operator](../../visual-basic/programming-guide/language-features/procedures/how-to-define-an-operator.md)
 - [How to: Define a Conversion Operator](../../visual-basic/programming-guide/language-features/procedures/how-to-define-a-conversion-operator.md)
-- [Object as the Universal Data Type (Visual Basic)](https://msdn.microsoft.com/library/5315bf21-2b22-45ab-98cd-5631dffbcb2f)
+- [Object as the Universal Data Type (Visual Basic)](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/3cf69sdx(v=vs.100))
```
Comment (mairaw) [inline docs/visual-basic/misc/bc33032.md]:
```suggestion
- [Object Data Type (Visual Basic)](../language-reference/data-types/object-data-type.md)
```

File: docs/visual-basic/misc/bc36553.md [hunk 3]
```diff
@@ -42,5 +42,5 @@ greeting.Print()
   
 ## See also
 - [Extension Methods](../../visual-basic/programming-guide/language-features/procedures/extension-methods.md)
-- [How to: Define Optional Parameters for a Procedure (Visual Basic)](https://msdn.microsoft.com/library/0b32b312-0da0-489b-96ad-7dcb1f1f8f88)
+- [How to: Define Optional Parameters for a Procedure (Visual Basic)](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/ssaskzyc(v=vs.100))
```
Comment (mairaw) [inline docs/visual-basic/misc/bc36553.md]:
```suggestion
```

File: docs/visual-basic/misc/bc36577.md [hunk 4]
```diff
@@ -19,5 +19,5 @@ An anonymous type initializer has been entered in the **Watch** or **Immediate**
   
 ## See also
 - [Anonymous Types](../../visual-basic/programming-guide/language-features/objects-and-classes/anonymous-types.md)
-- [Closer Look: What If... Testing Code in the Immediate Window](https://msdn.microsoft.com/library/3613a627-09a4-44e1-9cc2-f2a29f4e0744)
+- [Closer Look: What If... Testing Code in the Immediate Window](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2008/ms172593(v=vs.90))
```
Comment (mairaw) [inline docs/visual-basic/misc/bc36577.md]:
```suggestion
- [Immediate Window - Visual Studio](/visualstudio/ide/reference/immediate-window)
```

File: docs/visual-basic/misc/bc36595.md [hunk 5]
```diff
@@ -18,7 +18,7 @@ A method contains both an `On Error Goto` statement and either a lambda expressi
 1.  Replace the exception handling code that uses the `On Error Goto` statement with a `Try...Catch` statement.  
   
 ## See also
-- [Introduction to Exception Handling (Visual Basic)](https://msdn.microsoft.com/library/9792f16a-0cd2-40bd-ace2-f7a4344c0e52)
+- [Introduction to Exception Handling (Visual Basic)](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2010/c68xfk56(v=vs.100))
```
Comment (mairaw) [inline docs/visual-basic/misc/bc36595.md]:
```suggestion
- [Handling and throwing exceptions in .NET](../../dotnet/standard/exceptions/index.md)
```

File: docs/visual-basic/misc/bc36627.md [hunk 6]
```diff
@@ -19,5 +19,5 @@ A lambda expression has been entered in the **Watch** or **Immediate** window du
   
 ## See also
 - [Lambda Expressions](../../visual-basic/programming-guide/language-features/procedures/lambda-expressions.md)
-- [Closer Look: What If... Testing Code in the Immediate Window](https://msdn.microsoft.com/library/3613a627-09a4-44e1-9cc2-f2a29f4e0744)
+- [Closer Look: What If... Testing Code in the Immediate Window](https://docs.microsoft.com/previous-versions/visualstudio/visual-studio-2008/ms172593(v=vs.90))
```
Comment (mairaw) [inline docs/visual-basic/misc/bc36627.md]:
```suggestion
- [Immediate Window - Visual Studio](/visualstudio/ide/reference/immediate-window)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thank you @DennisLee-DennisLee. A few comments before we merge.

**Review by mairaw — APPROVED** [review]

**Pull request merged**